### DISCUSSION TITLE
如何实现在got中拒绝后让其他的事件响应器处理？

### DISCUSSION THREAD

**Discussion by Murasame2333** [nonebot/discussions#532] [问答] [answered]
如何实现在got中，如果收到的回复消息不符合预期(比如用户回复了一条指令)，会立刻结束当前的事件处理，并让其他的事件响应器进行处理？

**Comment by yanyongyu** [comment, marked as answer]
你可能需要在handle里将 matcher 的 block 属性设置为 False。
